src/types/login.ts

```diff
@@ -5,3 +5,7 @@ export interface LoginParams {
 export interface VerifyOtpParams extends LoginParams {
   otp: string
 }
+
+export interface GetSgidAuthUrlResponseDto {
```

Comment:
can we avoid putting http methods in the type?

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1369/commits/a14be908614d8a13dd3107dfbff4f5e276affcf1